spec/css/selector.hrx

```diff
@@ -55,6 +55,65 @@
   c: d;
 }
 
+<===>
+================================================================================
+<===> inline_comments/silent/input.sass
+a, // comment
+b
+  x: y
+
+<===> inline_comments/silent/output.css
+a,
+b {
+  x: y;
+}
+
+<===>
+================================================================================
+<===> inline_comments/silent/with_comma_in_comment/input.sass
+a // comment,
+b
+  x: y
```

Comment:
Consider [adding a comment](https://github.com/sass/sass-spec/blob/master/STYLE_GUIDE.md#consider-adding-a-comment-explaining-your-spec) explaining why this works the way it does.